docker/Dockerfile

```diff
@@ -0,0 +1,61 @@
+#
+# The docker image is built in two stages.  First stage builds the
+# oasis-node and oasis-core-rosetta-gateway binaries.
+# Second stage prepares the execution environment and copies the
+# two binaries over.
+#
+
+#
+# Build stage
+#
+FROM golang:1.13-alpine AS build
+
+# Install prerequisites.
+RUN rm -f /var/cache/apk/*; apk --no-cache add bash git make gcc g++ libressl-dev libseccomp-dev linux-headers
+
+ARG CORE_BRANCH="v20.6.1"
+ARG GATEWAY_BRANCH="master"
+
+# Fetch and build oasis-core.
+RUN git clone --single-branch --branch $CORE_BRANCH https://github.com/oasislabs/oasis-core /usr/local/build-core && \
+	cd /usr/local/build-core/go && \
+	make oasis-node && \
+	cp oasis-node/oasis-node /usr/bin/ && \
+	make clean && go clean -cache -testcache -modcache && \
+	cd / && rm -rf /usr/local/build-core
+
+# Fetch and build oasis-core-rosetta-gateway.
+RUN git clone --single-branch --branch $GATEWAY_BRANCH https://github.com/oasislabs/oasis-core-rosetta-gateway /usr/local/build-gateway && \
+	 cd /usr/local/build-gateway && \
+	 make && \
+	 cp oasis-core-rosetta-gateway /usr/bin/ && \
+	 make nuke && \
+	 cd / && rm -rf /usr/local/build-gateway
+
+
+#
+# Execution stage
+#
+FROM alpine:latest
+RUN rm -f /var/cache/apk/*; apk --no-cache add bash libressl libseccomp
+COPY --from=build /usr/bin/oasis-node /usr/bin/
+COPY --from=build /usr/bin/oasis-core-rosetta-gateway /usr/bin/
+
+VOLUME /data
+WORKDIR /data
+ENV OASIS_NODE_GRPC_ADDR="unix:/data/internal.sock"
+
+#
+# To prepare the /data directory, follow the instructions at:
```

Comment:
Mention that certain steps are not needed (e.g., there is no need to provision entities and nodes), basically you only need to get the correct `genesis.json` and a `config.yml` like the following:
```yaml
##
# Oasis Node Configuration
##

datadir: /data

log:
  level:
    # Per-module log levels. Longest prefix match will be taken. Fallback to
    # "default", if no match.
    default: debug
    tendermint: debug
    tendermint/context: error
  format: JSON

# Path to the genesis file for the current version of the network.
genesis:
  file: /data/etc/genesis.json

# Tendermint backend configuration.
tendermint:
  abci:
    prune:
      strategy: none
  core:
    listen_address: tcp://0.0.0.0:26656
  db:
    backend: badger
  p2p:
    seed:
      - "D14B9192C94F437E9FA92A755D3CC0341F2E87CF@34.82.86.53:26656"
```

Maybe include an example config somewhere in the repository? It could get stale though.